source/particle/world.cc

```diff
@@ -852,15 +970,199 @@ namespace aspect
 
                 // Only update particles, if there are any in this cell
                 if (particles_in_cell.begin() != particles_in_cell.end())
-                  local_update_particles(cell,
-                                         particles_in_cell.begin(),
-                                         particles_in_cell.end());
+                  {
+                    local_update_particles(cell,
+                                           particles_in_cell.begin(),
+                                           particles_in_cell.end()
+#if DEAL_II_VERSION_GTE(9,3,0)
+                                           , evaluators
+#endif
+                                          );
+                  }
+
               }
         }
     }
 
 
 
+#if DEAL_II_VERSION_GTE(9,3,0)
+    namespace internal
+    {
+      template <int dim>
+      class SolutionEvaluators
+      {
+        public:
+          SolutionEvaluators(const SimulatorAccess<dim> &simulator,
+                             const UpdateFlags update_flags)
+            :
+            velocity(simulator.get_mapping(),simulator.get_fe(), update_flags,simulator.introspection().component_indices.velocities[0]),
+            pressure(simulator.get_mapping(),simulator.get_fe(), update_flags,simulator.introspection().component_indices.pressure),
+            temperature(simulator.get_mapping(),simulator.get_fe(), update_flags,simulator.introspection().component_indices.temperature),
+            melt_component_indices(),
+            simulator_access(simulator)
+          {
```

Comment:
Can you move this and the other member functions of this class out of line as we usually do?

Reply:
I am not sure what you mean by out of line. I ran indent and broke up the long lines of arguments. Can you take another look if this is ok?

Reply:
I meant to separate the declaration from the implementation of the member functions of this class.

Reply:
Oh sure I can do that. I thought it saves some lines to keep it together since the class only lives inside this .cc file. Will do.